crates/teloxide-core/src/types/input_file.rs

```diff
@@ -1,10 +1,12 @@
 use bytes::{Bytes, BytesMut};
+#[cfg(not(target_family="wasm"))]
 use futures::{
     future::{ready, Either},
     stream,
 };
 use once_cell::sync::OnceCell;
 use rc_box::ArcBox;
+#[cfg(not(target_family="wasm"))]
```

Comment:
why?

Reply:
In the early tries the `Body.wrap_stream` is missing so I disabled these functions and removed the `into_part`